### DISCUSSION TITLE
How to use intune SDK in MAUI

### DISCUSSION THREAD

**Discussion by leungkimming** [dotnet/maui#12878] [Ideas]
When migrating Xamarin to MAUI, how to port Intune App SDK Xamarin Bindings to MAUI?

**Comment by KathirveluM** [comment]
We're still waiting for more details on the Intune SDK's support for the MAUI framework.

**Reply by leungkimming** [reply]
Please refer to Intune team's reply below
https://github.com/msintuneappsdk/intune-app-sdk-xamarin/issues/58#issuecomment-1420955192

**Comment by leungkimming** [comment]
I was told that Intune team have release a [beta version of our Android support for MAUI](https://www.nuget.org/packages/Microsoft.Intune.Maui.Essentials.android/9.5.2-beta).
